CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - `EXECUTION_RESOURCES` fields are hex-strings instead of integers
   - `segment_arena_builtin` resource is missing
   - v3 transaction price unit type is `STRK` instead of `FRI`
+- Transaction hashes for version 3 broadcasted transactions using the "query" flag in the version field are now computed correctly.
```

Comment:
We should agree on a consistent tone / phrasing for these "fix" notes. Either

1. note what was wrong (and we know it is now fixed due to it being under the fixed header), or
2. note that we are now doing something correctly

Example of 1, (combined under the v0.6 bullet since I think that's the only place?)
```suggestion
  - v3 transaction hashes are computed incorrectly when using the "query" flag, causing validation errors.
```